build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         javaTargetCompatibility = 8
         javaSourceCompatibility = 8
     }
-
+    ext['logback.version'] = '1.2.13'
```

Comment:
Since `1.2.13` is used in two places, is it worthwhile to use a variable for it?

Reply:
Not worth complicating the dependency section with embedded references.